packages/core/src/Modules/Cashier/Components/Error/deposit-locked.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { routes, getDerivComLink } from '@deriv/shared';
+import { Icon, Checklist } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { WS } from 'Services';
+
+const DepositsLocked = ({
+    account_status,
+    is_tnc_needed,
+    is_financial_information_incomplete,
+    is_trading_experience_incomplete,
+    is_financial_account,
+    history,
+    onMount,
+}) => {
+    // handle authentication locked
+    const { identity, document, needs_verification } = account_status.authentication;
+    const is_poi_needed = needs_verification.includes('identity');
+    const is_poa_needed = needs_verification.includes('document');
+    const has_poi_submitted = identity.status !== 'none';
+    const has_poa_submitted = document.status !== 'none';
+    const poi_text = has_poi_submitted
+        ? localize('Check proof of identity document verification status')
+        : localize('Upload an ID document to verify your identity');
+    const poa_text = has_poa_submitted
+        ? localize('Check proof of address document verification status')
+        : localize('Upload a proof of address to verify your address');
+
+    // handle TnC
+    const acceptTnc = async () => {
+        await WS.tncApproval();
+        await WS.getSettings();
+        onMount();
+    };
+
+    // handle all deposits lock status
+    const items = [
+        is_poi_needed && {
```

Comment:
Ah it's from here.

You can use this to prevent adding `false` to the array and remove the above check.

...(is_poi_needed  ? { content: poi_text, status: 'action', onClick } : [])

Reply:
i tried, and it gives null to the array, and it reads as empty item in the above check 🙏

Reply:
It does! The spread operator (...) for Array literals does nothing if its operand is an empty Array `[]`

Reply:
ya, you're right, i mistakenly put object literal directly instead of spreading array, let me try again, thank you!